apps/docs/src/.vitepress/components/OnyxIconLibrary.vue

```diff
@@ -37,7 +38,7 @@ const filteredCategories = computed(() => {
     <Search v-model="search" />
 
     <section v-for="category in filteredCategories" :key="category.name" class="category">
-      <h3 class="category__headline">{{ category.name }}</h3>
+      <OnyxHeadline is="h3" class="category__headline">{{ category.name }}</OnyxHeadline>
```

Comment:
they now are smaller: 20px is deployed, this here makes it be 16px font-size.
I asked Jannick which is correct

Reply:
https://www.figma.com/file/YfEUBOHk4J4nYrk04geswG/onyx---Design-System-Figma-Library?node-id=1288%3A31098&mode=dev

Reply:
16px/1rem is correct :)